bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -295,6 +295,32 @@ impl Client {
         Ok(())
     }
 
+    /// Login with JWT
+    pub async fn login_with_jwt(
+        &self,
+        jwt: String,
+        initial_device_name: Option<String>,
+        device_id: Option<String>,
+    ) -> Result<(), ClientError> {
+
+        let data: Map<String, Value> = [("token".to_owned(), jwt.into())]
+            .into_iter()
+            .collect();
```

Comment:
You could also write this as `let data = json!({ "token": jwt }).as_object().unwrap().clone();` which might be a bit cleaner.

Reply:
sure will make these changes